src/rules/gamma_inverse/out.jl

```diff
@@ -0,0 +1,9 @@
+export rule
+
+# Belief propagation
+# case: inbound messages on edges [α, θ] sum-product
+@rule GammaInverse(:out, Marginalisation) (m_α::Any, m_θ::Any) = GammaInverse(mean(m_α), mean(m_θ))
```

Comment:
This belief-propagation rule is correct only if both arguments are `PointMass` (remember that for belief propagation we generally do not use `Any`, if you spot somewhere most probably it is a mistake. For VMP in contrast `Any` is OK)
```suggestion
@rule GammaInverse(:out, Marginalisation) (m_α::PointMass, m_θ::PointMass) = GammaInverse(mean(m_α), mean(m_θ))
```